detectable/src/test/java/com/synopsys/integration/detectable/detectables/go/vendr/parse/VndrParserTest.java

```diff
@@ -0,0 +1,71 @@
+package com.synopsys.integration.detectable.detectables.go.vendr.parse;
+
+import java.util.Arrays;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import com.synopsys.integration.bdio.graph.DependencyGraph;
+import com.synopsys.integration.bdio.model.Forge;
+import com.synopsys.integration.bdio.model.externalid.ExternalIdFactory;
+import com.synopsys.integration.detectable.util.graph.NameVersionGraphAssert;
+
+class VndrParserTest {
```

Comment:
This is kind of a duplicate of the `GoVndrDetectableTest` class.
The parser converts straight to a `DependencyGraph`, but I'd prefer the parsers to be tested individually.
I think tests for lower level classes like parsers should be exhaustive of possibilities, while Detectable tests don't have to be.

Reply:
In this case there happens to be a lot of overlap.